sql/mozfun/vpn/channel_group/udf.sql

```diff
@@ -0,0 +1,109 @@
+CREATE OR REPLACE FUNCTION vpn.channel_group(
```

Comment:
in a later PR this will replace the `website_channel` group in `all_subscriptions_v1` and derived tables.